### PR TITLE
Add argonaut support

### PR DESCRIPTION
**Pull request by bonifaido**
I accept the the project's `Contribution policy`.
Could you please create a new release from the library when this gets merged?

### CODE REVIEW

File: akka-http-argonaut/src/main/scala/de/heikoseeberger/akkahttpargonaut/ArgonautSupport.scala [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package de.heikoseeberger.akkahttpargonaut
+
+import akka.http.scaladsl.marshalling.{ Marshaller, ToEntityMarshaller }
+import akka.http.scaladsl.model.{ HttpCharsets, MediaTypes }
+import akka.http.scaladsl.unmarshalling.{ FromEntityUnmarshaller, Unmarshaller }
+import argonaut._
```
Comment (hseeberger) [inline akka-http-argonaut/src/main/scala/de/heikoseeberger/akkahttpargonaut/ArgonautSupport.scala]: No wildcard imports at top-level, please.

**Comment by hseeberger** [comment]
Fantastic. Just one nitpick, please change, then I can merge.

**Comment by bonifaido** [comment]
Hm, somehow I picked an older argonaut version, let me update it to the latest.

**Comment by hseeberger** [comment]
Please collapse all commits into one "Add argonaut support"

**Comment by bonifaido** [comment]
And done.

**Comment by hseeberger** [comment]
Woot! Thanks a lot!

**Comment by hseeberger** [comment]
Released v1.4.2.

**Comment by bonifaido** [comment]
Fantastic! Thanks a lot!

**Pull request merged**